patterns/1-initial/document-architecture-decisions.md

```diff
@@ -0,0 +1,207 @@
+- [ADR-Tools from Nat Pryce - markdown](https://github.com/npryce/adr-tools)
+- [ADR-Tools from Christoph Kappel - asciidoc](https://github.com/unexist/adr-tools)
+- [Record-Tools from Christoph Kappel - acsiidoc](https://github.com/unexist/record-tools)
+- [mADR - markdown](https://adr.github.io/madr/)
+
+## Resulting Context
+
+Implementing an ADR/TDR-like process has proven to be valuable, as it makes architecture decision-making more transparent for everyone and ensures that all voices are heard.
+
+Observable positive effects:
+
+- **Democratizing architectural decision-making** to ensure that changes impacting multiple teams are inclusive, while also reducing the burden on team leads.
+- **Facilitating open, asynchronous communication** across teams and geographies, enhancing accessibility and collaboration. Contribute to ADR and TDR documents and engage in discussions to invite broader participation through writing workshops.
+- **Empowering individuals and teams** to take ownership of architectural decisions, enabling large-scale organizational change.
+- **Establishing a reliable record** of architectural decisions and technical debt, serving as a valuable reference for context and future decision-making.
+- **Maximizing the impact of experienced engineers** by enabling asynchronous and remote contributions, reducing the need for synchronous meetings.
+- **Standardizing terminology** by explicitly defining key concepts (e.g., "system tests") to ensure a shared understanding across teams.
+- **Aligning processes** by clearly documenting workflows such as the out-of-hours support procedure, promoting consistency and clarity.
+- **Enhancing clarity** of thought through documentation, encouraging authors to critically assess and refine their reasoning behind architectural decisions.
+
+The ADR/TDR approach also carries risks that a team want to acknowledge:
+
+- **It doesn’t always work!** Some people might still challenge a decision made through the ADR/TDR process. However, having a decision documented in writing is still valuable in these situations because it provides a clear record of when and why a decision was made.
+- **Pre-documenting design proposals (architecture, protocols, etc.) can resemble waterfall-like design**, which may not fit the iterative development model preferred by many teams. A team must remember the Agile principle: “Working software over comprehensive documentation.” Therefore, the ADR/TDR process should be as lightweight as possible to avoid unnecessary overhead.
+- **The ADR/TDR can become too long and cumbersome**, leading to long comment threads and debates. In these cases, a team might complement the process with synchronous communication, such as working groups or in-person meetings. Still, time is saved by allowing participants to review the ADR/TDR before the meeting rather than explaining everything during the discussion.
+
+## Rationale
+
+??
+
+## Known Instances
```

Comment:
Known Instances are organizations that have used this pattern (and have publicly shared somewhere that they do so).

Given that definition, some of the links below should rather be moved to the Related Material section.

e.g. from a quick scan it looks like the link [GitHub: ADR GitHub organization](https://adr.github.io/) tries to evangelize the use of ADRs. However it does not look like it describes the use of ADRs in the InnerSource-way at a specific organization, does it?

You know these resources a lot better than I do. Could you double check which of these we should treat as Known Instances, and what would rather be Related Material?

Reply:
You are right only two links (spotify and sap) are really reference a know instance. All other explain ADR usage or motivate to support it:) I move the links to related material section.